### DISCUSSION TITLE
Use flutter_background_service or flutter_foreground_service

### DISCUSSION THREAD

**Discussion by whowillcare** [gdelataillade/alarm#42] [Ideas]
Do you have any plan intergrating with thoese packages to make the alarm more robust in certain sense?

thanks

**Comment by gdelataillade** [comment]
Hi @whowillcare 

Good questions. I did some research and it looks like `flutter_background_service` uses `Background Fetch` for iOS, which do periodic background fetches every 15 minutes, which is not viable for an alarm.

What do you think ?

**Reply by whowillcare** [reply]
I know it's really a pain for iOS,  :( things always go south if it's in the background. 

I've been doing research as well and came acrossed your project, which is really great.

Have you tried the native EKAlarm or something more reliable Alarm package? Otherwise, the background_service or local notification might be a backup solution.

**Comment by whowillcare** [comment]
I know it's really a pain for iOS,  :( things always go south if it's in the background. 

I've been doing research as well and came acrossed your project, which is really great.

Have you tried the native EKAlarm or something more reliable Alarm package? Otherwise, the background_service or local notification might be a backup solution.

**Comment by gdelataillade** [comment]
Hi @whowillcare,

I finally found some time to take a look at `EKAlarm`. It looks like it's only used for Calendar and Reminders, not traditional alarms.

Thanks for the suggestion though, I really appreciate !
